web/src/app/schedules/util.js

```diff
@@ -30,6 +30,15 @@ export function mapRuleTZ(fromTZ, toTZ, rule) {
   }
 }
 
+export function clockToISO(clockTime, zone) {
+  return DateTime.fromFormat(clockTime, 'HH:mm', { zone })
+    .toUTC()
+    .toISO()
+}
+export function isoToClock(isoTime, zone) {
```

Comment:
Could this perhaps be renamed to `isoToMilitary` or s/t? The `Clock` keyword confused me at first and I had to look at the def. here to understand

Reply:
Hmm. We are using `Clock` as the type on the backend to represent `HH:mm` values, so I didn't think twice about it. But I can see how it'd be confusing (I think that's actually why Go calls it "Kitchen" time when AM/PM is involved).

I'll try thinking of a different way of naming it since all the types are similar. A literal name for this would be `ISOTimestampToISOTimeWithoutOffset` so maybe `timeFromISO` and `isoFromTime`?

Using "from" instead of "to" might be better as well now that I think about it because it loses the date information (e.g. not a 1-to-1 conversion).

Reply:
Might be worth noting, we also use `ClockTime` as the type in the GraphQL schema too... maybe we should stick with clock and just make sure to add comments (should have been there anyway)?

Reply:
I see. The only thing that concerns me is it's a data type that we have defined on the schema/backend, so context is inherently required to understand what this function might do to newcomers, as opposed to a more generally understood term. I understand where you're coming from though

Reply:
Actually, they do -- these functions are *only* used for converting to/from the GraphQL data type to and from the backend into ISO timestamps behind the scenes of the UI. I think that's where some of the confusion is stemming from.

Maybe `isoToGQLClockTime` and `gqlClockTimeToISO`? That way it's clear it's intent is for the backend API and not any type of formatting/use.